apps/src/storage/dataBrowser/DataTableView.jsx

```diff
@@ -22,6 +23,7 @@ const INITIAL_STATE = {
 class DataTableView extends React.Component {
   static propTypes = {
     // from redux state
+    isRtl: PropTypes.bool.isRequired,
```

Comment:
Out of curiosity is the `isRtl` pattern an existing one we already implement elsewhere to handle RTL languages?

Reply:
yeah, it's all over the place unfortunately.